editor/gui/editor_quick_open_dialog.cpp

```diff
@@ -226,6 +228,13 @@ QuickOpenResultContainer::QuickOpenResultContainer() {
 			grid->add_theme_constant_override("h_separation", 4);
 			grid->hide();
 			scroll_container->add_child(grid);
+
+			file_context_menu = memnew(PopupMenu);
+			file_context_menu->add_item(TTR("Show in FileSystem"), FILE_SHOW_IN_FILESYSTEM);
+			file_context_menu->add_item(TTR("Show in External File Explorer"), FILE_SHOW_IN_EXPLORER);
```

Comment:
```suggestion
			file_context_menu->add_item(TTR("Show in File Manager"), FILE_SHOW_IN_EXPLORER);
```
Consistent with FileSystem dock's option.